src/lib/dvf/delete-generic.js

```diff
@@ -0,0 +1,16 @@
+const { del } = require('request-promise')
+const _omitBy = require('lodash/omitBy')
+const _isNil = require('lodash/isNil')
+
+module.exports = async (dvf, endpoint, json = {}, headers = {}) => {
+  const url = dvf.config.api + endpoint
+
+  const options = {
+    uri: url,
+    headers,
+    // removes null and undefined values
+    json: _omitBy(json, _isNil)
```

Comment:
DELETE requests should not have a body so let’s not provide this option here

Reply:
Removed